### DISCUSSION TITLE
Direct discount

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15841] [Feature improvement]
It seems to be overlooked that direct discount cannot be applied to line items. It is kind of common I believe. So under 'Custom Discount' not only discount in percentage but also Direct Discount in decimal field would be nice :-)
Of course I mean when all line items are set in group discount

Best

**Comment by Marco31Bike** [comment]
Vtiger from where YF was born and other CRM/ERP systems have ability to add Direct Discount. It means that you can enter just value in certain currency. Was it overlooked or there is other reason to not implement it?
